src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/dropdown.scss

```diff
@@ -0,0 +1,49 @@
+@import '../Application/colors.scss';
+
+$dropdownBackgroundColor: $wildSand;
+$itemTextActiveColor: $shakespeare;
+
+.container {
+    display: none;
+    position: absolute;
+    left: 0;
+    top: 100%;
+    z-index: 1;
+    padding-right: 20px;
+    padding-left: 52px;
+    white-space: nowrap;
+    box-shadow: 0 0px 6px rgba(50, 50, 50, .25);
```

Comment:
The shadow color has already been defined in the toolbar, maybe use the same color variable here? (at least the one from `colors.scss`)